packages/matrix/package.json

```diff
@@ -22,6 +22,7 @@
     "start:admin": "docker start synapse-admin",
     "start:host-pre-built": "cd ../host && pnpm start --path ./dist",
     "stop:admin": "docker stop synapse-admin",
+    "register-test-user": "USERNAME=user PASSWORD=password ts-node --transpileOnly ./scripts/register-test-user.ts",
```

Comment:
A new command to register test user. It is documented here https://github.com/cardstack/boxel/pull/872/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R46.